docs/auditors/privesc.md

```diff
@@ -0,0 +1,69 @@
+# Privilege Escalation Allowed Auditor (privesc)
+
+Finds containers that allow privilege escalation.
+
+## General Usage
+
+```
+kubeaudit privesc [flags]
+```
+
+See [Global Flags](/README.md#global-flags)
+
+## Examples
+
+```
+$ kubeaudit privesc -f "auditors/privesc/fixtures/allow_privilege_escalation_nil_v1.yml"
+ERRO[0000] allowPrivilegeEscalation not set which allows privilege escalation. It should be set to 'false'.  AuditResultName=AllowPrivilegeEscalationNil Container=fakeContainerAPE
+```
+
+## Explanation
+
+`allowPrivilegeEscalation` controls whether a process can gain more privileges than its parent process.
+
+Privilege escalation is disabled by setting `allowPrivilegeEscalation` to `false` in the container SecurityContext. The field defaults to `true` so it must be explicitly set to `false` to pass the `privesc` audit:
```

Comment:
```
By default, ie when allowPrivilegeEscalation=nil, we will set no_new_privs=true with the following exceptions:

when a container is privileged
when CAP_SYS_ADMIN is added to a container
when a container is not run as root, uid 0 (to prevent breaking suid binaries)
```
this is complicated 😄

Reply:
Was that a copy paste from somewhere? What is `no_new_privs`?

Reply:
Oh I just have went down a rabbit hole with the documentation you linked 🙂

Reply:
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/no-new-privs.md

Reply:
Ah I see. Do you think we should change the documentation to capture those details?

Reply:
Ah I see. Do you think we should include more detail in our documentation?

Reply:
Nah I was just interested in how it works :)